fury/molecular.py

```diff
@@ -0,0 +1,749 @@
+        radius_type.
+
+        Parameters
+        ----------
+        atomic_number : int
+            Atomic number of the element whose covalent radius is to be
+            obtained.
+        radius_type : string
+            Two valid choices:
+            * 'VDW' : for Van Der Waals radius of the atom
+            * 'Covalent' : for covalent radius of the atom
+            Default: 'VDW'
+        """
+        radius_type = radius_type.lower()
+        if radius_type == 'vdw':
+            return self.GetVDWRadius(atomic_number)
+        elif radius_type == 'covalent':
+            return self.GetCovalentRadius(atomic_number)
+        else:
+            raise ValueError('Incorrect radius_type specified. Please choose'
+                             ' from "VDW" or "Covalent".')
+
+    def atom_color(self, atomic_number):
+        """Given an atomic number, return the RGB tuple associated with that
+        element (CPK coloring convention) provided by the Blue Obelisk Data
+        Repository.
+
+        Parameters
+        ----------
+        atomicNumber : int
+            Atomic number of the element whose RGB tuple is to be obtained.
+        """
+        rgb = np.array(self.GetDefaultRGBTuple(atomic_number))
+        return rgb
+
+
+def sphere_cpk(molecule, colormode='discrete'):
+    """Create an actor for sphere molecular representation. It's also referred
+    to as CPK model and space-filling model.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to be rendered.
+    colormode : string, optional
+        Set the colormode for coloring the atoms. Two valid color modes:
+        * 'discrete': Atoms are colored using CPK coloring convention.
+        * 'single': All atoms are colored with same color(grey).
+
+        RGB tuple used for coloring the atoms when 'single' colormode is
+        selected: (150, 150, 150)
+        Default: 'discrete'.
+
+    Returns
+    -------
+    molecule_actor : vtkActor
+        Actor created to render the space filling representation of the
+        molecule to be visualized.
+    """
+    colormode = colormode.lower()
+    msp_mapper = dcovtk.vtkOpenGLMoleculeMapper()
+    msp_mapper.SetInputData(molecule)
+    msp_mapper.SetRenderAtoms(True)
+    msp_mapper.SetRenderBonds(False)
+    msp_mapper.SetAtomicRadiusTypeToVDWRadius()
+    msp_mapper.SetAtomicRadiusScaleFactor(1)
+    if colormode == 'discrete':
+        msp_mapper.SetAtomColorMode(1)
+    elif colormode == 'single':
+        msp_mapper.SetAtomColorMode(0)
+    else:
+        msp_mapper.SetAtomColorMode(1)
+        warnings.warn("Incorrect colormode specified! Using discrete.")
+
+    # To-Do manipulate shading properties to make it look aesthetic
+    molecule_actor = rcvtk.vtkActor()
+    molecule_actor.SetMapper(msp_mapper)
+    return molecule_actor
+
+
+def ball_stick(molecule, colormode='discrete',
+               atom_scale_factor=0.3, bond_thickness=0.1,
+               multiple_bonds='on'):
```

Comment:
Why not just use a boolean here? True/False

Reply:
Done. Makes sense. Nice suggestion!